### PR TITLE
Add `is_global_ultimate` to API endpoints

### PR DESCRIPTION
**Pull request by currycoder**
### Description of change

This PR exposes `is_global_ultimate` on the relevant company API endpoints.

### Checklist

* [X] Has a new newsfragment been created? Check [changelog/README.md](https://github.com/uktrade/data-hub-leeloo/blob/master/changelog/README.md) for instructions
* [ ] Do any added or updated endpoints appear in the API documentation? See [docs/Maintaining the API documentation.md](https://github.com/uktrade/data-hub-leeloo/blob/develop/docs/Maintaining&#32;the&#32;API&#32;documentation.md) for more details
* [ ] Have any relevant search models been updated?
* [ ] Have any relevant fixtures (`fixtures/test_data.yaml`) been updated?
* [ ] Have any relevant select-/prefetch-related field lists in the views and search apps been updated?
* [ ] Has the admin site been updated (for new models, fields etc.)?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

File: datahub/company/test/test_company_views.py [hunk 1]
```diff
@@ -330,6 +330,7 @@ def test_get(self):
             'transferred_to': None,
             'transfer_reason': '',
             'pending_dnb_investigation': False,
+            'is_global_ultimate': False,
```
Comment (alixedi) [inline datahub/company/test/test_company_views.py]: `company.is_global_ultimate`?

File: datahub/company/serializers.py [hunk 2]
```diff
@@ -443,6 +443,7 @@ class Meta:
             'pending_dnb_investigation',
             'export_potential',
             'great_profile_status',
+            'is_global_ultimate',
```
Comment (alixedi) [inline datahub/company/serializers.py]: This should be in `read_only_fields`. Probably `dnb_read_only_fields`.

File: datahub/dnb_api/test/test_views.py [hunk 3]
```diff
@@ -387,6 +387,9 @@ def _assert_companies_same(self, company, dnb_company):
             'county': dnb_company.get('registered_address_county') or '',
             'postcode': dnb_company.get('registered_address_postcode') or '',
         } if required_registered_address_fields_present else None
+        is_global_ultimate = True if (
+            dnb_company['global_ultimate_duns_number'] == dnb_company['duns_number']
+        ) else False
```
Comment (alixedi) [inline datahub/dnb_api/test/test_views.py]: How about:

```
is_global_ultimate = dnb_company['global_ultimate_duns_number'] == dnb_company['duns_number']
```

or inline (see next comment).

File: datahub/dnb_api/test/test_views.py [hunk 4]
```diff
@@ -439,6 +442,7 @@ def _assert_companies_same(self, company, dnb_company):
             'transferred_on': None,
             'contacts': [],
             'pending_dnb_investigation': False,
+            'is_global_ultimate': is_global_ultimate,
```
Comment (alixedi) [inline datahub/dnb_api/test/test_views.py]:
```
'is_global_ultimate': (
    dnb_company['global_ultimate_duns_number'] == dnb_company['duns_number']
)
```

File: datahub/company/serializers.py (hunk 2 above)
Comment (currycoder) [reply datahub/company/serializers.py]: It's in `read_only_fields` already.  Not in `dnb_read_only_fields` as they are an extension to `read_only_fields` for DNB matched companies.

File: datahub/dnb_api/test/test_views.py (hunk 3 above)
Comment (currycoder) [reply datahub/dnb_api/test/test_views.py]: Who knows why I wrote it the first way - thanks!

File: datahub/company/serializers.py (hunk 2 above)
Comment (alixedi) [reply datahub/company/serializers.py]: Didn't realise that it needed to be in `fields` as well as `read_only_fields` 😬

File: changelog/company/add-is-global-ultimate-property.api.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+The property `is_global_ultimate` (a boolean) was added as a field to the following company API
```
Comment (reupen) [inline changelog/company/add-is-global-ultimate-property.api.md]: Would be good to clarify that it's read-only (and also, in the case of `POST /v4/dnb/company-create`, it's in the response).

**Pull request merged**